keyboards/neodox/readme.md

```diff
@@ -0,0 +1,21 @@
+# The Neodox keyboard
+
+Inspired by Redox from Mattia Dal Ben
+
+**Neodox** project. More information and building instruction [here](todo).
+
+- Keyboard Maintainer: [MatteoDM](todo)  
+- Hardware Supported: rp2040 ,ws2812b, GC9A01, [handwire 3D printed case](https://www.thingiverse.com/thing:5625196)
+- Hardware Availability: handwire at the moment
```

Comment:
This whole project should be moved to the `handwired` folder.

Reply:
moved into handwired